src/Lemon/Containers/Converters/NodeContainer2BinaryCia.cs

```diff
@@ -77,11 +78,25 @@ public BinaryFormat Convert(NodeContainerFormat source)
             };
 
             Node root = source.Root;
+
+            var title = root.Children["title"].GetFormatAs<TitleMetadata>();
```

Comment:
In case we modified the `title` node from a file, the format may not be always `TitleMetadata` but `BinaryFormat`. I suggest we do a check here and convert if needed. Something like:
```csharp
if (root.Children["title"].Format is IBinary) {
    root.Children["title"].TransformWith<Binary2TitleMetadata>();
}

if (root.Children["title"].Format is not TitleMetadata) {
    throw new FormatException("Unknown format for title node");
}

var title = root.Children["title"].GetFormatAs<TitleMetadata>();
```

With this change, we don't need to convert in `Binary2Cia` anymore.